tests/List/SelectionSpec.elm

```diff
@@ -106,4 +113,26 @@ spec =
                         |> Selection.mapSelected { selected = (*) 2, rest = identity }
                         |> Expect.equal items
             ]
+        , describe "filtering"
+            [ fuzz (selection Fuzz.int) "works like regular filter on lists" <|
+                \items ->
+                    items
+                        |> Selection.toList
+                        |> List.filter isEven
+                        |> Expect.equal (Selection.filter isEven items |> Selection.toList)
+            , fuzz (nonemptySelection Fuzz.int) "preserves selected item" <|
+                \( item, items ) ->
```

Comment:
possibly just multiply the item by 2 and get rid of the case? This test is a little confusing to read, so it's definitely going to get stale.